src/lib/rng/jitter_rng/jitter_rng.cpp

```diff
@@ -0,0 +1,103 @@
+/*
+* CPU Jitter Random Number Generator
+* (C) 2024 Planck Security S.A.
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <botan/jitter_rng.h>
+
+#include <jitterentropy.h>
+
+namespace Botan {
+
+class BOTAN_DLL Jitter_RNG_Data {
+   public:
+      Jitter_RNG_Data();
```

Comment:
This part would then change to:
```
class BOTAN_DLL Jitter_RNG::Data {
   public:
      Data();
```
Same for the rest of places where the name of the structure is written

Reply:
A side comment, just so there are no misunderstandings:
I'm not that familiar with this repo, so my suggestions may not be the usual or preferred thing to do here, it's just what I consider correct or helpful.
My words are not authoritative here. Best regards :)